File: January 5, 2026.js
function tireStatus(pressuresPSI, rangeBar) {

  const tiresStatus = [];
  const rangePSI = [rangeBar[0] * 14.5038, rangeBar[1] * 14.5038];
  
  for (let tire of pressuresPSI) {
    if (tire > rangePSI[1]) {
      tiresStatus.push("High");
    } else if (tire > rangePSI[0]) {
      tiresStatus.push("Good");
    } else {
      tiresStatus.push("Low");
    }
  }

  return tiresStatus;
}

File: January 5, 2026.py
def tire_status(pressures_psi, range_bar):

    tires_status = []
    range_psi = [range_bar[0] * 14.5038, range_bar[1] * 14.5038]

    for tire in pressures_psi:
        if tire > range_psi[1]:
            tires_status.append("High")
        elif tire > range_psi[0]:
            tires_status.append("Good")
        else:
            tires_status.append("Low")

    return tires_status